Monika After Story/game/cardgames.rpy

```diff
@@ -0,0 +1,4781 @@
+                and persistent._mas_game_nou_wins["Monika"] > persistent._mas_game_nou_wins["Player"]
+            ):
+                m "I'm sure if we play more games you'll win too."
+
+            else:
+                "Maybe next time you'll win~"
+
+    elif choice == 2:
+        m "I won the game too!"
+        m "Ehehe~"
+        m "Thanks for playing with me, [player]."
+
+    else:
+        m "And I'm the first who reached [persistent._mas_game_nou_house_rules[win_points]] points!"
+        m "I won this time~"
+    return
+
+label mas_nou_reaction_player_surrenders:
+    $ persistent._mas_game_nou_wins["Abandoned"] += 1
+    $ store.mas_nou.player_win_streak = 0
+    # TODO: consider silently add points to Monika here
+    # so you can't abuse it when you're losing
+
+    if persistent._mas_game_nou_wins["Abandoned"] > 4:
+        m "That's alright, [player]..."
+        m "But promise you'll finish the game next time?{w=0.4} For me?~"
+
+    elif persistent._mas_game_nou_wins["Abandoned"] > 2:
+        m "[player]...{w=0.3}{nw}"
+        extend "you keep giving up on our games..."
+        m "I hope you're enjoying playing with me."
+        m "I do enjoy every moment I'm with you~"
+
+    elif store.mas_nou.game.total_turns == 1:
+        m "But we just started."
+        m "Let me know when you'll have time."
+
+    elif store.mas_nou.game.total_turns < 6:
+        m "Giving up already, [player]?"
+        if (
+            len(store.mas_nou.game.monika.hand) < 5
+            and len(store.mas_nou.game.player.hand) > 8
+        ):
+            m "I love to play with you no matter what the outcome is!"
+            m "I hope you're feeling the same way."
+
+        else:
+            m "You could at least try..."
+
+    else:
+        # This part isn't really correct, but she just tries to support you
+        if len(store.mas_nou.game.monika.hand) >= len(store.mas_nou.game.player.hand):
+            m "I'm pretty sure you could win this [_round!t], [player]!"
+
+        else:
+            if len(store.mas_nou.game.monika.hand) > 1:
+                m "Actually I had quite bad cards, [player]."
+            else:
+                m "Actually I had a quite bad card, [player]."
+
+            m "I think you could win this [_round!t]."
+
+        m "Don't give up so easily next time."
+    return
+
+label mas_nou_reaction_monika_reflected_act(seen_count=0):
+    if seen_count == 0:
+        m "I read you as an open book~"
+
+    elif seen_count == 1:
+        m "Nope!"
+
+    else:
+        m "Still nope~"
+    jump mas_nou_game_loop
+
+label mas_nou_reaction_player_reflected_act(seen_count=0):
+    m "Just this time..."
+    jump mas_nou_game_loop
+
+label mas_nou_reaction_monika_reflected_wdf(seen_count=0):
+    m "Ahaha! I knew you will do that!"
+    jump mas_nou_game_loop
+
+label mas_nou_reaction_player_reflected_wdf(seen_count=0):
+    m "Fuck."
```

Comment:
So, I am not a writer, and am quite bad at writing Monika's "voice". So this isn't a recommendation, so much as my thoughts. But this would be the first serious curse word in the mod if I am not mistaken. Monika does not really cuss and this seems off to me. Then again, Monika is not a dainty flower, and knows the word fuck. I'd like to see some other opinions on this and maybe suggestions on a better way to show her frustrations. I dunno tho *shrug*